pydrad/configure/configure.py

```diff
@@ -415,7 +425,8 @@ def maximum_cells(self):
         refinement level and :math:`n_{min}` is the minimum allowed number of
         grid cells. Optionally, if the maximum number of cells is specified
         in ``config['grid']['maximum_cells']``, this value will take
-        precedence.
+        precedence. In general, it is better to set this value explicitly to a sufficiently
+        large value.
```

Comment:
What is sufficiently large?  Perhaps 1 Mm per cell at the minimum?  I think a simple example number would be useful as a comment.

Reply:
The default configuration now explicitly uses a value of 30000 which is what the HYDRAD GUI uses. Beyond that, I don't have a sense of what sufficiently large is. For low refinement levels (e.g. 4), the $2^Ln_{min}$ rule of thumb does not seem to hold.

Reply:
I changed this to say "30,000" instead of "sufficiently large" as that is the value used by the HYDRAD GUI as well as the value in the default config file in `pydrad`.